src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -491,7 +491,7 @@ private static string TryFindFile(string xmlFilePath, string uriString = null)
             internal static OpenFileCommand DeserializeCore(XmlElement element)
             {
                 XmlElementHelper helper = new XmlElementHelper(element);
-                string xmlFilePath = TryFindFile(helper.ReadString("XmlFilePath"), element.OwnerDocument.BaseURI);
+                string xmlFilePath = TryFindFile(helper.ReadString("FilePath"), element.OwnerDocument.BaseURI);
```

Comment:
After doing some debugging this code is not hit during XML deserialization inside Dynamo usng the sandbox. I agree with your above comment and so have set the attribute name here back to XmlFilePath